src/input/mod.rs

```diff
@@ -152,22 +150,30 @@ impl<'a> InputFile<'a> {
 
 
     /// Reads image data into the given FrameBufferMut.
+    ///
+    /// The passed FrameBufferMut must match the image's resolution exactly, and
```

Comment:
Hmm... so this is more general than just this method.  It's true for all read/write methods.  So I'm thinking maybe the note actually belongs in the documentation for FrameBuffer(Mut), which is the common denominator.  Does that seem reasonable to you?